src/page/bank-payment.page.ts

```diff
@@ -4,7 +4,10 @@ export class BankPaymentPage {
   private confirmOrder: ElementFinder;
 
   constructor () {
-    this.confirmOrder = $('#cart_navigation > button > span');
+    // this.confirmOrder = $('#cart_navigation > button > span');
+    // I changed the selector because in the code block of #cart_navigation
+    // is only one 'span', so whit those two thing we can identidy the element that we need.
+    this.confirmOrder = $('#cart_navigation span');
```

Comment:
I think that this doesn't make any difference with the original one, you can play a little bit more, try to use a different locator, for example, the text related with the button seems to be unique "I confirm my order". Find a locator in where you can use that text.

Reply:
until the moment, i have not find yet a locator that works, i will keep trying.

Reply:
what about to use the protractor functions instead?